bgpd/bgp_route.c

```diff
@@ -10014,19 +10008,49 @@ DEFUN (show_ip_bgp,
 		return bgp_show_route_map(vty, bgp, argv[idx + 1]->arg, afi,
 					  safi, bgp_show_type_route_map);
 
+	/* prefix-longer */
+	if (argv_find(argv, argc, "A.B.C.D/M", &idx)
+	    || argv_find(argv, argc, "X:X::X:X/M", &idx))
```

Comment:
why not just search for "longer-prefixes"?

Reply:
Dunno, I just moved the whole block to the newly created `cmd`.